xlators/mgmt/glusterd/src/glusterd-handshake.c

```diff
@@ -1296,15 +1282,12 @@ gd_validate_mgmt_hndsk_req(rpcsvc_request_t *req, dict_t *dict)
         0,
     };
     glusterd_peerinfo_t *peer = NULL;
-    xlator_t *this = NULL;
+    xlator_t *this = THIS;
```

Comment:
nit: just pass this from the single caller of this function - __glusterd_mgmt_hndsk_versions()  (remember to make this function static)

Reply:
Ack.